report_thesis/src/sections/problem_definition.tex

```diff
@@ -1,23 +1,60 @@
 \section{Problem Definition}\label{sec:problem_definition}
+The primary objective of this research is to enhance computational methods for the accurate and robust quantification of chemical compositions using \gls{libs} data.
+This enhancement seeks to improve the adaptability and precision of \gls{libs} across various environmental contexts, including terrestrial and planetary geological samples.
 
-\subsection{Motivating Example: NASA's Mars Missions}
-The NASA Viking missions in the 1970s were the first to successfully land on Mars, aiming to determine if life existed on the planet. 
-While these missions advanced our knowledge of the Martian environment, the search for evidence of life remained inconclusive~\cite{marsnasagov_vikings}.
+Quantifying chemical compositions from \gls{libs} spectral data poses significant challenges due to the high dimensionality of the data and the complex physical and chemical interactions involved. The interpretation of \gls{libs} data is particularly complicated by factors such as multicollinearity among spectral features and matrix effects, which can obscure the relationships between the observed spectra and the actual chemical compositions.
```

Comment:
We've mentioned multicollinearity and matrix effects in the introduction. This sentence reads a little like its the first time we've presented it. Does it perhaps make sense to rephrase it so that its clear that its not the first time we mention it?